blog/index.tt

```diff
@@ -1526,7 +1526,7 @@ for the next edition or look at the
 <p>lorri is based around fast direnv integration for robust CLI and editor integration.</p>
 <p><a href="http://github.com/target/lorri">Try it out</a></p>
 <ul>
-<li><a href="https://nixos.org/releases/patchelf/patchelf-0.10/">PatchELF 0.10 released</a></li>
+<li><a href="https://releases.nixos.org/?prefix=patchelf/patchelf-0.10/">PatchELF 0.10 released</a></li>
```

Comment:
I get `Error: [object Object]` here

Reply:
Strange, the link works for me in firefox on macos. [https://releases.nixos.org/?prefix=patchelf/patchelf-0.10/](https://releases.nixos.org/?prefix=patchelf/patchelf-0.10/). 

<img width="1492" alt="Screenshot 2021-12-11 at 09 59 00" src="https://user-images.githubusercontent.com/8530224/145672467-339574fc-3fb9-42c5-8ce9-0632e7840b56.png">

Reply:
I was caused by uBlock, sorry for the noise.